### DISCUSSION TITLE
Discussions!

### DISCUSSION THREAD

**Discussion by WakuWakuPadoru** [WakuWakuPadoru/WuWa_Simple_FPSUnlocker#1] [Announcements]
If you have any questions or issues, don't hesitate to start a new discussion or raise a new issue!

**Comment by JackDemon316** [comment]
Hay i just tried your new release and it works but for some reason my controller is not being detected. i was wondering if you can test it on your end to see if this is just a my issue plz?

**Reply by WakuWakuPadoru** [reply]
Hello, thanks for the message. I've been playing the game with a wired Xbox Controller and haven't run into any issue. There seems to be a bug in the game where if you have a controller that's connected both wirelessly and wired, the game won't detect it so nothing much I can do if it's a game problem.

If you're connected vie Bluetooth, I'd recommend turning off your Bluetooth and connecting your controller wired to see if it works.

https://game8.co/games/Wuthering-Waves/archives/453141

**Reply by JackDemon316** [reply]
i just delete the driver and that fixed the issue ty for respond tho.

**Reply by WakuWakuPadoru** [reply]
No problem! Enjoy :)

**Comment by CighiMihai** [comment]
Works perfectly for me! Note to anyone that wants to use this and gets the "attempt to write on a read-only database" select the right exe for the game, I was a bonobo in the beginning and I selected the client's exe not the game's :)
PS: Thank you very much for making this unlocker, you are the best!

**Comment by hhrocha000** [comment]
after apply this i get a "beep/bip" sound a couple of times through my play session, is this normal? i get freezes too when i get those beeps/bips

**Comment by WakuWakuPadoru** [comment]
> after apply this i get a "beep/bip" sound a couple of times through my play session, is this normal? i get freezes too when i get those beeps/bips

This is not normal and usually indicates some sort of hardware issue (doesn't mean that it's faulty).

The beep sounds are a way for your computer to tell you that it cannot handle your inputs. For example if it is struggling to handle the higher FPS and hardware load.

What you can do is:
1) Shut down your PC and turn it back on. (Not a restart)
2) Update your Graphics Drivers.
3) Reduce settings, either the FPS and/or the Graphics settings.

There are people who get this issue even without using any unlocks so this could just be a game optimisation issue :( [Link here](https://www.reddit.com/r/WutheringWaves/s/bARp2feWoH)

**Reply by 27rogi** [reply]
It seems like some inner game issue for sure, I also get those stupid beeping sounds when clicking too fast and game kinda freezes at this time. Effect is similar to that Windows is using for sticky keys (like SHIFT), but it seems that this is built-in game feature 😕

**Reply by WakuWakuPadoru** [reply]
> It seems like some inner game issue for sure, I also get those stupid beeping sounds when clicking too fast and game kinda freezes at this time. Effect is similar to that Windows is using for sticky keys (like SHIFT), but it seems that this is built-in game feature 😕

The beeping sounds in this context lets you know that the game or WIndows is struggling to process your inputs such as key strokes or mouse presses etc. Your PC could be working too hard or the game might be having problems managing your PC's resources such as your memory. This isn't too uncommon of a problem so seems like a game issue :)

**Reply by 27rogi** [reply]
> > It seems like some inner game issue for sure, I also get those stupid beeping sounds when clicking too fast and game kinda freezes at this time. Effect is similar to that Windows is using for sticky keys (like SHIFT), but it seems that this is built-in game feature 😕
> 
> The beeping sounds in this context lets you know that the game or WIndows is struggling to process your inputs such as key strokes or mouse presses etc. Your PC could be working too hard or the game might be having problems managing your PC's resources such as your memory. This isn't too uncommon of a problem so seems like a game issue :)

I honestly doubt that RTX 4060, 7735HS and 16GB of RAM is not enough for this game lol, so even if it's having overload, it's due to bad optimization :P

**Comment by Imperial-Egineer** [comment]
This is indeed a great work. Just needs more polishing I know you guys can do it. Thank you for this. The stuttering issue still persist in some way. In the case when the game loaded first/initial walk stutters. Then it will eventually smooth ups after a few sec. I think this indeed deep in game related optimization issues. The fps unlocks does truly work!  This is way better than the freagin 60 fps. My rig can handle other games (world of warships and etc. at high or ultra high settings)without any issues but dont know for this game somehow its cranky. My specs R5 5600g, Rtx 2060, 16gb 3200mhz (2x8) game installed in SSD m.2 nvme. The game graphics still really need some in depth reworks.
